sdk/ml/azure-ai-ml/azure/ai/ml/_schema/component/data_transfer_component.py

```diff
@@ -38,6 +38,15 @@ class DataTransferCopyComponentSchema(DataTransferComponentSchemaMixin):
         values=NestedField(InputPortSchema),
     )
 
+    @validates("outputs")
+    def outputs_key(self, value):
+        outputs_count = len(value)
+        if outputs_count != 1:
+            msg = "Only support single output in {}, but there're {} outputs."
```

Comment:
maybe print output name list instead of count only

Reply:
get output name of outputs here is hack. I think count is enough here